guides/_subsections/navigate-the-dashboard.md

```diff
@@ -14,67 +14,67 @@ Dashboard cards are widgets that display information about your Cryostat instanc
 <ol>
   <li>
     {% capture target-jvm-details-text %}
-    The <i>Target JVM Details</i> card displays information about the target JVM application that is currently selected. There are two tabs that display different information:
+    The <i>Target JVM Details</i> <code>card</code> displays information about the <code>target</code> <b>JVM</b> application that is currently selected. There are two tabs that display different information:
```

Comment:
"Add the <i>Target <b>JVM</b> Details</i> <code>Card</code>" in lines 54 and 57. Honestly think it would be better to change those two - I don't think we need the JVM to be bolded when you're talking about the Target JVM Details card.

Reply:
Also capital C here for Card